submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+
+### There's a mod for it
+
+Speaking of Minecraft, it's popularity boom in 2013 and with it one of the largest modding communities ever,[^2][^3] it was most definitely the first introduction to technical gameplay - and to some degree even automation - to many players, especially younger ones. But that got me thinking: With mods in the likes of Buildcraft, IndustrialCraft, and Redpower featuring a very heavy focus on industrial machinery and automation, and release dates well before Factorio's development even started,[^4] could that have been an inspiration for Factorio?
+
+Wube's online presents, while there, is spread out across the forums, reddit, discord, and a few more, which makes it difficult to search for anything, really. But after some clever searching and sifting through a few interviews, I finally found  something: An [old forum thread](https://forums.factorio.com/viewtopic.php?f=5&t=3026) from March 2014, confirming Buildcraft (or IndustrialCraft, according to a different interview) as the main inspiration to Factorio.[^5][^6] While I had hoped to find an answer, I for sure wasn't expecting to find such a direct one. Having that out of the way, though, we can definitely speculate a bit more on brought Factorio to were it's at today.
+
+### Great games design themselves
+
+There is a principle in game design you should follow, that goes along the lines "follow the fun".[^7] While examples for that statement often include dramatic changes like Ape Out went through,[^8] it still applies to games that don't shift that drastically in direction. Let's take Factorio itself as an example: <!-- The example should be commented by devs, and an actually relevant feature | Options I'm considering: Pickaxe, Character Tab --> Originally the game had multiple pickaxes - a wooden, which you could then craft into a better one out of stone, and then into the iron pickaxe we had for such a long time [^citation needed]. This was clearly inspired by Minecraft (and also confirmed by the devs[^9]), but having so many items about manual mining in a game about automation gives the idea about the game - it draws the fun away. So they simplified it, gave you just your iron and steel pickaxe, but that's still two items specifically about manual mining. Thus they removed pickaxes completely from the game in `0.17.0`[^10], and with all focus on the unfun, manual labor.
+
+So once they core gameplay concept of a game is clear - which is what Factorio started out with - it's just a manner of fitting features to it. Automation? That sounds like the industrial revolution, when they invented belts. Need something to get that thing of the belts? How about inserters, they are at least not human. You get the point.
```

Comment:
This pararaph is a little unclear and could use a little refining. But I see you made a comment about it :)